physics/noahmptable.tbl

```diff
@@ -0,0 +1,780 @@
+&noahmp_usgs_veg_categories
```

Comment:
Not sure where this data file belongs? @grantfirl @Qingfu-Liu Thoughts?

Reply:
Good question. Stuff like this is typically kept outside of the physics source code, somewhere in an input directory for the host model. Other than meta files, this would be the first ascii-formatted file in the ccpp-physics repo if this is accepted. I think that supermodule (fv3atm and ufs-weather-model) code managers should probably weigh in. @jkbk2004 @junwang-noaa @BrianCurtis-NOAA

Reply:
Those data are collected together from some of the NOAH-MP FORTRAN code. I feel it OK to be here, but like to know other's opinions.

Reply:
Note that this parameter file is a bit of a temporary solution. After this PR, we would like to shift to using noahmp as a submodule. The parameter file will exist in the external repository, [for example](https://github.com/NCAR/noahmp/blob/master/parameters/NoahmpTable.TBL).